cmd/timestamp-server/app/serve.go

```diff
@@ -80,11 +81,29 @@ var serveCmd = &cobra.Command{
 		port := int(viper.GetUint("port"))
 		scheme := viper.GetStringSlice("scheme")
 
+		ntpMonitoring := viper.GetString("ntp-monitoring")
+		var ntpm *ntpmonitor.NTPMonitor
+		if ntpMonitoring != "" {
+			log.Logger.Infof("ntp monitoring: %s", ntpMonitoring)
+			go func() {
+				ntpm, err = ntpmonitor.New(ntpMonitoring)
+				if err != nil {
+					log.Logger.Errorf("error initializing ntp monitor %s", err)
+					return
```

Comment:
It'd be good to call `Fatalf` here to halt initialization since this is just on server startup